flagd/tests/integration/json_evaluator_test.go

```diff
@@ -0,0 +1,36 @@
+package integration_test
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/cucumber/godog"
+	flagd "github.com/open-feature/go-sdk-contrib/providers/flagd/pkg"
```

Comment:
Minor remark - this brings a circular dependency between flagd & flagd provider of go. I think this is not impacting us as we only use `ProviderOption` which is not changing much. May be something we can improve if we have the bandwidth to focus on the improvement. Otherwise let's create a follow-up improvement story to track :)

Reply:
We already had such an cycle (the `evaluation` and `caching` tests worked the same way) but I see your point.